validation-test/stdlib/Set.swift

```diff
@@ -1175,16 +1157,16 @@ SetTestSuite.test("deleteChainCollisionRandomized") {
         return k
       }
     }
-    let hashValue = uniformRandom(chainLength - chainOverlap) * collisionChains
+    let hashValue = Int.random(in: 0 ..< (chainLength - chainOverlap), using: &generator) * collisionChains
     let k = TestKeyTy(value: value, hashValue: hashValue)
     knownKeys += [k]
     return k
   }
 
   var s = Set<TestKeyTy>(minimumCapacity: 30)
   for i in 1..<300 {
-    let key = getKey(uniformRandom(collisionChains * chainLength))
-    if uniformRandom(chainLength * 2) == 0 {
+    let key = getKey(Int.random(in: 0 ..< (collisionChains * chainLength), using: &generator))
+    if Int.random(in: 0 ..< (chainLength * 2), using: &generator) == 0 {
       s.remove(key)
     } else {
       s.insert(TestKeyTy(key.value))
```

Comment:
@lorentey Is this correct?

* `SetTestSuite` is creating a new `TestKeyTy` for each insertion (with a matching `value` and `hashValue`).
* `DictionaryTestSuite` reuses the `TestKeyTy` from `getKey(_:)` (with a separately randomized `hashValue`).

Reply:
It sure doesn't look correct. I believe the `Set` tests should also insert the key from `getKey(_:)`. I'll review & fix in a followup.